From 6a6deba682976ff795eb613833565c446dc19ff4 Mon Sep 17 00:00:00 2001
From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
Date: Wed, 29 Mar 2023 16:06:01 +0530
Subject: [PATCH] Revert "Config Changes for enabling iwl7000 module"

This reverts commit aac742407e50663bbd04c6db26ed118d29017d64.
Ported changes form lts2019-chromium to linux-intel-lts2021

Tracked-On:
Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>

diff --git a/groups/wlan/iwlwifi/init.rc b/groups/wlan/iwlwifi/init.rc
index 29e29d7..5741beb 100644
--- a/groups/wlan/iwlwifi/init.rc
+++ b/groups/wlan/iwlwifi/init.rc
@@ -8,7 +8,7 @@ on early-boot
     insmod /vendor/lib/modules/compat.ko
 {{/iwl_upstream_drv}} 
     insmod /vendor/lib/modules/cfg80211.ko
-    insmod /vendor/lib/modules/iwl7000_mac80211.ko
+    insmod /vendor/lib/modules/mac80211.ko
     insmod /vendor/lib/modules/iwlwifi.ko
     insmod /vendor/lib/modules/iwlmvm.ko power_scheme=1
 
-- 
2.40.0

